test/unit/scripting_spec.js

```diff
@@ -361,15 +361,15 @@ describe("Scripting", function () {
         name: "Keystroke",
         willCommit: false,
         change: "o",
-        selStart: 4,
-        selEnd: 4,
+        selStart: -1,
```

Comment:
All the changes in this file are very likely wrong.

Reply:
These changes may be wrong in that they are against spec, but they are correct in that they are what pdf.js is emitting currently. It sounds like pdf.js's behavior in creating these events is the actual bug though.

Reply:
So to be clear, I agree these will probably need to end up being reverted, hopefully replaced by some tests (or fixed test) of the generation of events for single-character edits, to validate that they are populating `selStart`/`selEnd` like this further up in the process.